app/controllers/concerns/hal_actions/actions.rb

```diff
@@ -42,6 +42,8 @@ def destroy_redirect
   def index_options
     valid_params_for_action(:index).tap do |options|
       options[:_keys] = options.keys
+      # TODO: this isn't great
+      options[:represent_with] = Api::PagedCollectionRepresenter
```

Comment:
For sure, that is the plan, and how I would expect things to work to maintain consistency. Now, if it ceases to be the plan, a single controller could overwrite this method, or we could modify when we get there.